java-scala/project/BuildSettings.scala

```diff
@@ -24,8 +24,8 @@ object BuildSettings {
     organization  := "com.snowplowanalytics",
     version       := "0.3.0",
     description   := "Library for extracting marketing attribution data from referer URLs",
-    scalaVersion  := "2.9.1",
-    crossScalaVersions := Seq("2.9.1", "2.10.4", "2.11.1"),
+    scalaVersion  := "2.11.11",
+    crossScalaVersions := Seq("2.9.1", "2.10.4", "2.11.11"),
```

Comment:
I think we can ditch 2.9.1 at this point, and bump to 2.10.6.

Reply:
Completely forget I changed this. Was mainly so I could get to work for myself as I couldn't get it to compile at all otherwise. Let me know if this needs changing.

Reply:
yup let's do:

```scala
crossScalaVersions := Seq("2.10.6", "2.11.11"),
```